Client/Frontend/Browser/FindInPageBar.swift

```diff
@@ -0,0 +1,136 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import UIKit
+
+protocol FindInPageBarDelegate: class {
+    func findInPage(findInPage: FindInPageBar, didTextChange text: String)
+    func findInPage(findInPage: FindInPageBar, didFindPreviousWithText text: String)
+    func findInPage(findInPage: FindInPageBar, didFindNextWithText text: String)
+    func findInPageDidPressClose(findInPage: FindInPageBar)
+}
+
+class FindInPageBar: UIView {
+    weak var delegate: FindInPageBarDelegate?
+    private let searchText = UITextField()
+    private let matchCountView = UILabel()
+    private let previousButton = UIButton()
+    private let nextButton = UIButton()
+
+    var currentResult = 0 {
+        didSet {
+            matchCountView.text = "\(currentResult)/\(totalResults)"
+        }
+    }
+
+    var totalResults = 0 {
+        didSet {
+            matchCountView.text = "\(currentResult)/\(totalResults)"
+            previousButton.enabled = totalResults > 1
+            nextButton.enabled = previousButton.enabled
+        }
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        backgroundColor = UIColor.whiteColor()
+
+        searchText.addTarget(self, action: "didTextChange:", forControlEvents: UIControlEvents.EditingChanged)
```

Comment:
Yeah, I actually started this convention way back when we first started the project. The goal was to make it obvious that these methods are selectors to prevent us from accidentally removing them when we do cleanup (since there won't be any compile-time errors). This was before we knew about the `@objc` annotation, though, which is usually a good indicator that the method is a selector (along with the `sender` argument).

I'll go ahead and add the prefix here just to follow our current convention. I can bring this up in the next iOS meeting to see if people still find it useful.